src/esp/gfx/PbrDrawable.cpp

```diff
@@ -419,6 +424,51 @@ void PbrDrawable::draw(const Mn::Matrix4& transformationMatrix,
     shader_->setTextureMatrix(matCache.textureMatrix);
   }
 
+  // clearcoat data
+  if (flags_ & PbrShader::Flag::ClearCoatLayer) {
```

Comment:
Maybe just change all flag `&`s to `>=`? Unnecessary inconsistency that makes the reader look twice, and prone to accidents if some of the flags would be multiple bits.